tests/test_remove_mon_from_cluster.py

```diff
@@ -0,0 +1,108 @@
+"""
+A Testcase to remove mon from
+when I/O's are happening.
+
+Polarion-ID- OCS-355
+
+"""
+
+import logging
+import re
+import random
+import time
+import pytest
+from ocs_ci.ocs import ocp, constants
+from ocs_ci.framework.testlib import tier4, ManageTest
+from ocs_ci.framework import config
+from ocs_ci.ocs.resources import pod
+from tests.helpers import run_io_with_rados_bench, delete_cephblockpool
+from ocs_ci.ocs.cluster import CephCluster
+
+
+log = logging.getLogger(__name__)
+
+DEP = ocp.OCP(
+    kind=constants.DEPLOYMENT, namespace=config.ENV_DATA['cluster_namespace']
+)
+POD = ocp.OCP(
+    kind=constants.POD, namespace=config.ENV_DATA['cluster_namespace']
+)
+
+
+def get_mons_from_cluster():
```

Comment:
Is it worth to move this function to https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/cluster.py?